torch/csrc/autograd/variable.h

```diff
@@ -576,6 +581,23 @@ inline Variable make_variable(
   return Variable();
 }
 
+// Moves out of the tensor instead of copying it.
+// Can save ~10μs of overhead.
```

Comment:
I think this comment should be put in the function declaration above instead, to be in line with other factory functions. Also we likely shouldn't mention the actual number of overhead time savings because it's test environment dependent and also subject to change in implementation.

Reply:
Thanks, that's a good point.